.kokoro/common.cfg

```diff
@@ -17,21 +17,14 @@
 # Download secrets from Cloud Storage.
 gfile_resources: "/bigstore/cloud-devrel-kokoro-resources/java-docs-samples"
 
-# Download trampoline resources. 
+# Download trampoline resources.
 gfile_resources: "/bigstore/cloud-devrel-kokoro-resources/trampoline"
 
 # Use the trampoline script to run in docker.
 build_file: "java-docs-samples/.kokoro/trampoline.sh"
```

Comment:
From the looks of the test failure, this might have to move to the `common.cfg` in the sub-directories? I wonder if the Build configs don't inherit the same way that the job configs do?

Reply:
I see a note in the Kokoro docs for Build Config "For build config, common.cfg must be put under the same directory as your build config."